src/adapters/surveyListAdapter.js

```diff
@@ -0,0 +1,11 @@
+import Api from 'utils/Api';
+
+const SurveyListAdapter = (page, token) => {
```

Comment:
@rafayet-monon  How about changing this to a class `SurveyAdapter` so we can reuse it on the API call for survey detail?

```js
class SurveyAdapter {
  static getList = (page, token) => {
    return Api.get(`api/v1/surveys?page[number]=${page}`, {
      headers: {
        Authorization: token
      },
    });
  };

  static get = (id) => {
    ...
  };
}

export default SurveyAdapter;
```

and use it like this

```js
await SurveyAdapter.getList(1, token)

await SurveyAdapter.get(surveyID, token)
```

Reply:
Edited in [18e6a16](https://github.com/rafayet-monon/surveyor/pull/17/commits/18e6a16ac7f0831897e820c423760f575692c9f6) along with the height problem.